svg.js.d.ts

```diff
@@ -951,7 +952,7 @@ declare namespace svgjs {
 
         rotate(degrees: number, cx?: number, cy?: number): Animation
         skew(x: number, y: number): Animation
-        scale(x: number, y: number): Animation
+        scale(scaleX: number, scaleY: number, cx?: number, cy?: number): Animation

```

Comment:
`scale` may called with one parameter only. So `scaleY` has to be optional, too. Please change that bit and I guess we can merge that. Thanks for your contribution!